src/Microsoft.Health.Dicom.Core.UnitTests/Features/Workitem/WorkitemServiceTests.cs

```diff
@@ -50,7 +50,7 @@ public WorkitemServiceTests()
     public async Task GivenNullDicomDataset_WhenProcessed_ThenArgumentNullExceptionIsThrown()
     {
         await Assert.ThrowsAsync<ArgumentNullException>(
-            async () => await _target.ProcessAddAsync(null, string.Empty, CancellationToken.None).ConfigureAwait(false));
```

Comment:
Depending on this return type, you may be able to remove both `async` and `await` here